db/migrate/20230317183337_add_confirmation_fields_to_users.rb

```diff
@@ -0,0 +1,7 @@
+class AddConfirmationFieldsToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :confirmed_at, :datetime
+    add_column :users, :confirmation_token, :string
+    add_column :users, :confirmed_sent_at, :datetime
+  end
+end
```

Comment:
- [ ] **[OPTIONAL]** You should consider deleting this migration file as it throws errors. These columns have already been added by Devise, so we don't need them.

  ![image](https://user-images.githubusercontent.com/54848363/226101570-d4d326dc-ca6f-4504-aaeb-e548c033959c.png)